script/seed.js

```diff
@@ -39,7 +93,7 @@ if (module === require.main) {
     console.error(err)
     process.exitCode = 1
   })
-  .finally(() => { // `finally` is like then + catch. It runs no matter what.
+  .then(() => { // `finally` is like then + catch. It runs no matter what.
```

Comment:
Why did you need to do this? `finally` works on my machine!

_this is an example code review comment._